daemons/compute/server/servers/server_default.go

```diff
@@ -247,7 +256,136 @@ func (s *defaultServer) Create(ctx context.Context, req *pb.DataMovementCreateRe
 		return nil, err
 	}
 
-	source, err := s.findRabbitRelativeSource(ctx, req)
+	computeClientMount, computeMountInfo, err := s.findComputeMountInfo(ctx, req)
+	if err != nil {
+		return &pb.DataMovementCreateResponse{
+			Status:  pb.DataMovementCreateResponse_FAILED,
+			Message: err.Error(),
+		}, nil
+	}
+
+	response := &pb.DataMovementCreateResponse{}
+
+	if computeMountInfo.Type == "lustre" {
+		response, err = s.createNnfDataMovement(ctx, req, userId, groupId, computeClientMount)
```

Comment:
You can just `return s.create*` here in both the if and else - then get rid of the response variable.